sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/CosmosRowConverterSpec.scala

```diff
@@ -423,6 +456,42 @@ class CosmosRowConverterSpec extends UnitSpec with BasicLoggingTrait {
     objectNode.get(colName4).asInt() shouldEqual colVal3
   }
 
+  "date and time in spark row" should "should honor dateTimeConversionMode config" in {
```

Comment:
This test only test the `convertSparkDataTypeToJsonNodeNonNull`
can we also add a test for `convertSparkDataTypeToJsonNodeConditionally`